src/lang/fr.json

```diff
@@ -1022,6 +1022,7 @@
   "user.name.label": "nom",
   "user.newsletterOptIn.description": "Abonnez-vous à la newsletter d'Open Collective.",
   "user.shortBio.label": "Courte description",
+  "user.since.label": "puisque",
```

Comment:
That's not correct, it's "depuis".

Reply:
Yes, sure. I fixed this one.

Reply:
@znarf I removed the translations and ran the script "./scripts/check_translations.sh".

Reply:
@znarf is there anything else you want to change?

Reply:
@leonardohof No, that was my only problem.